quantecon/dle.py

```diff
@@ -69,15 +69,14 @@ def __init__(self, information, technology, preferences):
         self.nb, self.nh = self.llambda.shape
         self.nd, self.nc = self.phic.shape
         self.nz, self.nw = self.c2.shape
-        junk, self.ng = self.phig.shape
+        self.junk, self.ng = self.phig.shape
```

Comment:
I would say this should be

```suggestion
        _, self.ng = self.phig.shape
```
```

Reply:
Thank you for pointing this out. I updated the PR accordingly.